vocabulary/gbif/agent_role.xml

```diff
@@ -219,4 +219,28 @@
     <alternative>
     </alternative>
   </concept>
+  <concept dc:identifier="reviewer" dc:URI="http://rs.gbif.org/vocabulary/gbif/agentRole/reviewer"
+           dc:relation=""
+           dc:description="a certified agent responsible for examining the dataset and providing assurance that it fulfills all quality requirements defined by GBIF.">
+    <preferred>
+      <term dc:title="Reviewer" xml:lang="en"/>
+      <term dc:title="Verificador" xml:lang="es"/>
+      <term dc:title="Vérificateur" xml:lang="fr"/>
+      <term dc:title="審閱者" xml:lang="zh"/>
+    </preferred>
+    <alternative>
+    </alternative>
+  </concept>
+  <concept dc:identifier="mentor" dc:URI="http://rs.gbif.org/vocabulary/gbif/agentRole/mentor"
+           dc:relation=""
+           dc:description="an agent who is highly experienced in publishing biodiversity data and who has provided guidance and constructive feedback during the publishing process.">

```

Comment:
Ideally a suitable mentor would have been awarded a biodiversity publishing badge for example. I didn't want to limit it to only mentors with badges though..